### PR TITLE
Optional token expiration after period of inactivity

### PR DESCRIPTION
**Pull request by riggy**
Goal of this PR is to add a feature of expiring the tokens after certain period of inactivity. As it was requested in this issue: https://github.com/adamniedzielski/tiddle/issues/34

### CODE REVIEW

File: spec/rails_app/config/application.rb [hunk 1]
```diff
@@ -1,10 +1,8 @@
 require File.expand_path('../boot', __FILE__)
 
 require "active_model/railtie"
-require "active_job/railtie"
```
Comment (riggy) [inline spec/rails_app/config/application.rb]: Pre-emptive comment: I had trouble even using the gem because of those lines here. I figured they weren't really needed so I removed them. I decided to keep it unless you want it back, so I'm gonna make another commit to roll this back.

**Comment by adamniedzielski** [comment]
Thanks a lot @riggy for the PR 💚 🥇. It looks good to me. I'll do some follow-up commits to make sure that the existing applications still work even when they don't have `expires_in` in their schema.
